src/modules/events-app.module/module.html

```diff
@@ -1,19 +1,23 @@
 {{ require_css(get_asset_url('../../event-registration.css')) }}
 {{ require_js(get_asset_url('../../event-registration.js'), 'footer') }}
 
-{% set IMAGES = {
-  'close': get_asset_url('../../images/close.png'),
-  'map': get_asset_url('../../images/map.png'),
-  'city': get_asset_url('../../images/city.jpg'),
+{% set APP_CONFIG = {
+  'appRoot': slug,
+  'portalId': portal_id,
+  'images': {
+    'close': get_asset_url('../../images/close.png'),
+    'map': get_asset_url('../../images/map.png'),
+    'city': get_asset_url('../../images/city.jpg'),
   }
+}
 %}
+{% require_js position="head" %}
+<script>
+  var APP_CONFIG = {{ APP_CONFIG|tojson }};
```

Comment:
@gcorne I'm curious if this is what you were thinking. If so, I'm not sure if it would be a better convention to attach this to `window`?